src/components/ControlPanel/controlPanel.js

```diff
@@ -31,18 +31,18 @@ export default class ControlPanelComponent extends React.Component {
 
 				<div className="form-elem">
 					<h1 className="DisplayPrimary">Event Analytics</h1>
-					<select className="Headline-2Secondary">
-						<option>General</option>
-						<option>AI</option>
+					<select 
+						className="Headline-2Secondary" >
 						<option>Board</option>
+						<option>AI</option>
 						<option>Hack</option>
 						<option>ICPC</option>
 						<option>NetSec</option>
 						<option>Studio</option>
 						<option>W</option>
 					</select>
-
 				</div>
+				<TimeSeries />
```

Comment:
Don't add this line yet. We'll add components to the main view when we connect everything together.